provider/aws/formation/rack.json

```diff
@@ -633,6 +633,11 @@
       "Description": "The security groups (comma delimited) to assign to the rack router.",
       "Type": "CommaDelimitedList"
     },
+    "ServiceScalingLogging": {
```

Comment:
Let's not even make this a rack param. If someone really needs to debug this function (unlikely) they can manually set the env var on the Lambda function.